examples/svg-layer.js

```diff
@@ -47,8 +52,21 @@ map.addLayer(
         center[1] / svgResolution - height / 2,
       );
       svgContainer.style.transform = cssTransform;
-      svgContainer.style.opacity = this.getOpacity();
+      svgContainer.style.opacity = this.getOpacity() || 1;
```

Comment:
Why the ` || 1`? 
`this.getOpacity()`  always returns 1 here; and what if it opacity is zero?

Reply:
This change avoids that the style ends up being the string `'undefined'`, which is not valid CSS.

Reply:
You were absolutely right, @MoonE . I undid the opacity change. Thanks for the sharp eye.